syntax = "proto3";

package zedstreamer;

service ZedStreamer{
  rpc SendImage(Image) returns (Received){}
  rpc SendVideo(stream Image) returns (Received){}
}

message Image {
  int32 width = 1;
  int32 height = 2;
  bytes image_data = 3;
}

message Received{
  bool ack = 1;
}